lib/l10n/intl_en.arb

```diff
@@ -59,7 +59,10 @@
 
   "warningInternetConnection": "Please make sure you are connected to the internet.",
   "warningPasswordLength": "The password must be 8 characters long or more.",
-  "warningPasswordCharacters": "The password must include lowercase and uppercase letters and at least one number and special character (!@#$&*~).",
+  "warningPasswordUppercase": "The password must include at least an uppercase letter.",
```

Comment:
Fă-le pe toate să zică "one" (one lowercase letter, one special character etc), for consistency.